package.json

```diff
@@ -256,19 +365,32 @@
       {
         "command": "cmake.outline.revealInCMakeLists",
         "title": "%cmake-tools.command.cmake.outline.revealInCMakeLists.title%"
+      },
+      {
+        "command": "cmake.folders.selectActiveFolder",
+        "title": "%cmake-tools.command.cmake.folders.selectActiveFolder.title%"
       }
     ],
     "menus": {
       "commandPalette": [
         {
           "command": "cmake.configure"
         },
+        {
+          "command": "cmake.configureAll"
```

Comment:
Does it make sense to have these "All" commands available when doing single root?  Can we add a "when" clause that enables these only when multi-root is active?  (use `util.setContextValue` to define a variable that can be checked here)

Reply:
I think somethingall = something under single root mode make more sense, so the user can avoid some extra work when writing tests for an extension depending on cmake tools.

Reply:
This just determines whether the command appears in the command palette or not. I believe extensions can still invoke the command (though we should check).